clientbuffer.cpp

```diff
@@ -52,9 +52,14 @@ class CClientBufferMod : public CModule
     MODCONSTRUCTOR(CClientBufferMod)
     {
         AddHelpCommand();
-        AddCommand("AddClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnAddClientCommand), "<identifier>", "Add a client.");
-        AddCommand("DelClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnDelClientCommand), "<identifier>", "Delete a client.");
-        AddCommand("ListClients", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnListClientsCommand), "", "List known clients.");
+        AddCommand("AddClient", t_d("<identifier> [timelimit]"), t_d("Add a client with optiona time limit."),
+                   [=](const CString& sLine) { OnAddClientCommand(sLine); });
```

Comment:
Could you please rationalize the use of C++ lambdas here? As I understand this introduces a dependency on newer versions of C++ compilers.

Reply:
I did that because most of modules provided out of the box with znc use that kind of construction. Besides that it doesn't matter for me which way you wanna go with that.

Reply:
> I did that because most of modules provided out of the box with znc

I'm guessing you're referring to the current version of ZNC. This module remains compatible with older versions, and I think that should be preserved unless a good reason comes up to break that compatibility.